example/cajita_tutorial/01_types/types_example.cpp

```diff
@@ -113,10 +113,11 @@ void typesExample()
 //---------------------------------------------------------------------------//
 int main( int argc, char* argv[] )
 {
-    Kokkos::ScopeGuard scope_guard( argc, argv );
-
-    typesExample();
+    {
+        Kokkos::ScopeGuard scope_guard( argc, argv );
 
+        typesExample();
+    }
```

Comment:
Why would you add a scope?

Reply:
I was copying from the MPI examples - I'll revert these